src/main/java/org/jabref/gui/LibraryTab.java

```diff
@@ -597,8 +597,8 @@ private void setupAutoCompletion() {
         if (autoCompletePreferences.shouldAutoComplete()) {
             suggestionProviders = new SuggestionProviders(getDatabase(), Globals.journalAbbreviationRepository, autoCompletePreferences);
         } else {
-            // Create suggestion providers with database for crossref if auto-completion is deactivated
-            suggestionProviders = new SuggestionProviders(getDatabase());
+            // Create empty suggestion providers if auto-completion is deactivated
+             suggestionProviders = new SuggestionProviders();
```

Comment:
Note that this reverts https://github.com/JabRef/jabref/pull/10829. The CHANGELOG.md needs to be updated then. Just remove the CHANGELOD.md entry.

Reply:
> Note that this reverts #10829. The CHANGELOG.md needs to be updated then. Just remove the CHANGELOD.md entry.

I think it will be still a valid entry.

Reply:
Ah, I see. Crossref will work even if auto complete is disabled. Nice.